components/net/http_loader.rs

```diff
@@ -500,6 +500,11 @@ pub fn http_fetch(
     *done_chan = None;
     // Step 1
     let mut response: Option<Response> = None;
+    context
+        .timing
+        .lock()
+        .unwrap()
+        .set_attribute(ResourceAttribute::FetchStart);
```

Comment:
We should initialize our fetchStart value in https://github.com/servo/servo/blob/49cf3dd3d337689ac72bea1a8ffc5c3e89a2616c/components/net/fetch/methods.rs#L90 instead, so it encompasses all kinds of requests.